packages/sync/src/Queue.php

```diff
@@ -455,46 +603,85 @@ private function get_next_data_row_option_name() {
 		return 'jpsq_' . $this->id . '-' . $timestamp . '-' . $this->random_int . '-' . $this->row_iterator;
 	}
 
+	/**
+	 * Return the items in the queue.
+	 *
+	 * @param null|int $limit Limit to the number of items we fetch at once.
+	 *
+	 * @return array|object|null
+	 */
 	private function fetch_items( $limit = null ) {
 		global $wpdb;
 
 		if ( $limit ) {
-			$query_sql = $wpdb->prepare( "SELECT option_name AS id, option_value AS value FROM $wpdb->options WHERE option_name LIKE %s ORDER BY option_name ASC LIMIT %d", "jpsq_{$this->id}-%", $limit );
+			$items = $wpdb->get_results(
+				$wpdb->prepare(
+					"SELECT option_name AS id, option_value AS value FROM $wpdb->options WHERE option_name LIKE %s ORDER BY option_name ASC LIMIT %d",
+					"jpsq_{$this->id}-%",
+					$limit
+				),
+				OBJECT
+			);
 		} else {
-			$query_sql = $wpdb->prepare( "SELECT option_name AS id, option_value AS value FROM $wpdb->options WHERE option_name LIKE %s ORDER BY option_name ASC", "jpsq_{$this->id}-%" );
+			$items = $wpdb->get_results(
+				$wpdb->prepare(
+					"SELECT option_name AS id, option_value AS value FROM $wpdb->options WHERE option_name LIKE %s ORDER BY option_name ASC",
+					"jpsq_{$this->id}-%"
+				),
+				OBJECT
+			);
 		}
 
-		return $this->query_for_items( $query_sql );
+		return $this->unserialze_values( $items );
+
 	}
 
+	/**
+	 * Return items with specific ids.
+	 *
+	 * @param array $items_ids Array of event ids.
+	 *
+	 * @return array|object|null
+	 */
 	private function fetch_items_by_id( $items_ids ) {
 		global $wpdb;
-		$ids_placeholders = implode( ', ', array_fill( 0, count( $items_ids ), '%s' ) );
-
-		$query_sql = $wpdb->prepare(
```

Comment:
@enejb  why change this one?

Reply:
phpcs complained and I wasn't able to ignore it successfully.

Reply:
complained about?